engines/osxnative/payload-schema.yml

```diff
@@ -0,0 +1,31 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Payload"
+type:               "object"
+description: |-
+  Payload for Mac OSX Engine
+properties:
+  link:
```

Comment:
So really I think that we should include any of this in the command part of the payload for now.  In the future we might want to look into doing something like this:

1. add task.payload.context to act similar to how .image is for docker.
    a. this context will be built from the decision task
    b. context lives in tree
    c. context will contain a single setup.sh script that will be invoked by the engine to setup the environment (hg clone, install packages, whatever)
    d. would look like task.payload.context = "https://queue.taskcluster.net/v1/1234/artifacts/public/contexts/osxnative/context.tar.gz"